packages/storybook/stories/profilePicture.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { ProfilePicture, sizeClasses } from '@dailydotdev/shared/src/components/ProfilePicture';
+import {  ProfilePictureGroup } from '@dailydotdev/shared/src/components/ProfilePictureGroup';
+
+// TODO: this story fails due to the below import which we will need to mock
+//       jest mocking is not currently supported in storybook
+// jest.mock('../hooks/useRequestProtocol', () => ({
+//   useRequestProtocol() {
+//     return {
+//       isCompanion: false,
+//     };
+//   },
+// }));
```

Comment:
Don't think we should push this to main, rather remove it or fix it :)

Reply:
It's actually breaking with `process` not defined. I am gonna remove it at this point and log.

Reply:
https://dailydotdev.atlassian.net/browse/WT-2038